docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -255,6 +255,12 @@ different than a Pandas timestamp. It is recommended to use Pandas time series f
 working with timestamps in `pandas_udf`s to get the best performance, see
 [here](https://pandas.pydata.org/pandas-docs/stable/timeseries.html) for details.
 
+### Recommended Pandas and PyArrow Versions
+
+For usage with pyspark.sql, the supported versions of Pandas is 0.24.2 and PyArrow is 0.15.1. Higher
```

Comment:
@BryanCutler, seems we should fix one instance at "Ensure PyArrow Installed" too.